spec/import-js/importer_spec.rb

```diff
@@ -273,6 +273,66 @@ def self.current_selection=(index)
         EOS
         end
       end
+
+      context 'with declaration_keyword' do
+        subject do
+          ImportJS::Importer.new.import
+          VIM::Buffer.current_buffer.to_s
+        end
+
+        let(:configuration) do
+          {
+            'declaration_keyword' => 'const'
+          }
+        end
+
+        context 'with a variable name that will resolve' do
+          let(:resolved_files) { ['bar/foo.js.jsx'] }
+
+          it 'adds an import to the top of the buffer using the declaration_keyword' do
+            expect(subject).to eq(<<-EOS.strip)
+const foo = require('bar/foo');
+
+foo
+            EOS
+          end
+
+          context 'when that variable is already imported using `var`' do
+            let(:text) { <<-EOS.strip }
+var foo = require('bar/foo');
+
+foo
+            EOS
+
+            it 'changes the `var` to declaration_keyword' do
+              expect(subject).to eq(<<-EOS.strip)
+const foo = require('bar/foo');
+
+foo
+              EOS
+            end
+          end
+
+          context 'when other imports exist' do
+            let(:text) { <<-EOS.strip }
+var zoo = require('foo/zoo');
+let bar = require('foo/bar');
+
+foo
+            EOS
+
+            it 'adds the import and sorts the entire list' do
+              expect(subject).to eq(<<-EOS.strip)
+const foo = require('bar/foo');
+let bar = require('foo/bar');
+var zoo = require('foo/zoo');
```

Comment:
I think that's exactly what I did, since alphabetizing the variable names would result in "bar, foo, zoo" no? Or maybe I don't understand your suggestion.